webapp/src/store/transfer-courses/reducers.ts

```diff
@@ -0,0 +1,35 @@
+import {GET_TRANSFER_COURSES_REQUEST, GET_TRANSFER_COURSES_SUCCESS, GET_TRANSFER_COURSES_ERROR, TransferCoursesActionTypes, TransferCoursesState} from './types';
```

Comment:
🤷 personally I'm fine with long lines since they wrap in my editor and don't scroll, but I can add a `max-len` rule. I don't have a strong opinion either way.

Reply:
ac05e22